### DISCUSSION TITLE
Scaledobjects or not found

### DISCUSSION THREAD

**Discussion by kumarnaveentulasi** [kedacore/keda#6402] [Q&A / Need Help]
Warning ScaledObjectCheckFailed 47m (x9 over 47m) keda-operator Target resource doesn't exist
Warning ScaledObjectCheckFailed 47m (x9 over 47m) keda-operator ScaledObject doesn't have correct scaleTargetRef specification
Normal KEDAScalersStarted 47m keda-operator Started scalers watch
Normal ScaledObjectReady 47m (x3 over 47m) keda-operator ScaledObject is ready for scaling
Warning KEDAScalerFailed 47m keda-operator error inspecting rabbitMQ: Exception (404) Reason: "NOT_FOUND - no queue 'inference_queue' in vhost '/'"
Normal KEDAScalersStarted 2m31s (x719 over 47m) keda-operator Scaler rabbitmq is built.

but my hpa was as follows

apiVersion: keda.sh/v1alpha1
kind: ScaledObject
metadata:
name: my-microservice-scaledobject
namespace: dev
spec:
scaleTargetRef:
name: my-microservice # The name of your deployment
pollingInterval: 5 # Time interval to poll RabbitMQ queue for messages
cooldownPeriod: 30 # Time to wait before scaling down after scaling up
minReplicaCount: 0 # Minimum replicas can scale to 0
maxReplicaCount: 20 # Maximum replicas (adjust based on your needs)
triggers:

type: rabbitmq
metadata:
host: "amqp://admin:Truefan@[123@ec2-3-108-9-50.ap-south-1.compute.amazonaws.com](mailto:123@ec2-3-108-9-50.ap-south-1.compute.amazonaws.com):5672" # Correct RabbitMQ URL
queueName: "inference_queue" # The RabbitMQ queue name to monitor
vhost: "naveen-test-dev" # The vhost name (ensure this is the correct vhost in RabbitMQ)
queueLength: "1" # Scale 1 pod per message in the queue
i was the file name as hpa.yml and the deploying in the dev namespace and keda operator in the keda namespace can any help me with this

**Comment by davogen** [comment]
Hi, I think this is the issue:

> Reason: "NOT_FOUND - no queue 'inference_queue' in vhost '/'"
